Controller/Adminhtml/IndexingManager/Reindex.php

```diff
@@ -30,34 +36,80 @@ public function __construct(
     /**
      * @throws ExceededRetriesException
      * @throws AlgoliaException
-     * @throws NoSuchEntityException
+     * @throws NoSuchEntityException|DiagnosticsException
      */
     public function execute()
     {
-        $storeIds = $this->getRequest()->getParam("store_id") === '0' ?
+        $params = $this->getRequest()->getParams();
+        $storeIds = !isset($params["store_id"]) || $params["store_id"] === '0' ?
```

Comment:
Can we be clearer here by using constants? e.g. `\Algolia\AlgoliaSearch\Service\AlgoliaConnector::ALGOLIA_DEFAULT_SCOPE`